version/version.go

```diff
@@ -0,0 +1,5 @@
+package version
+
+var (
+	Version = "0.9.8"
```

Comment:
Why don't you just hard code the `sdkVersion` value here? that way you can remove all the translation code.

Reply:
This is to keep the code more readable and maintainable by future developers. The sdkVersion reported to wavefront in its gauge format might be confusing to some.